tools/benchmark_against_libflac.sh

```diff
@@ -32,12 +32,12 @@ for i in {1..11}; do
     # overwrite.
     rm -f testsamples/extra/*.wav
     echo -n "[libflac]"
-    /bin/time --format="%e" --append --output /tmp/bench_times_libflac.dat \
+    $(which time) --format="%e" --append --output /tmp/bench_times_libflac.dat \
```

Comment:
When I run `which time` in my shell I get `time: shell reserved word` (which I why I put in `/bin/time` in here). I think the proper way to make this portable is to use `/usr/bin/env time`. Alternatively you can just put `/usr/bin/time` in there; on my system `/bin`/ is a symlink to `/usr/bin`.

Reply:
Good call. I've switched it to `env time`.